### PR TITLE
Check if any LGPL docs sneaked in

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Oh nice! Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -50,12 +50,15 @@ before_install:
...
   - rustc --version
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
+  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.14" ]; then
     make regen_check;
     fi
   - ./check_init_asserts
-  - cargo doc --features "dox"
-  - cargo test --features "$FEATURES"
+  - cargo doc --features "dox,embed-lgpl-docs"
+  - cargo test --features "$FEATURES,embed-lgpl-docs"
```
Comment (EPashkin) [inline .travis.yml]: I still slightly worry about updating all *.rs on second `embed-lgpl-docs` but 👍 anyway

Comment (sdroege) [reply .travis.yml]: That's how it is in the other `.travis.yml`. See https://github.com/gtk-rs/pango/pull/136#discussion_r255456569

**Comment by sdroege** [comment]
@GuillaumeGomez all green here otherwise

**Pull request merged**